en/flight_modes/return.md

```diff
@@ -59,26 +78,20 @@ If no rally points are defined, this is the same as a _Return to Launch_ (RTL)/_
 
 ### Mission Landing/Rally Point Return Type (RTL_TYPE=1)
 
+This is the default return type for a [fixed-wing](../flight_modes_fw/return.md) or [VTOL](../flight_modes_vtol/return.md) vehicle (see topics for more information).
+
 In this return type the vehicle:
 
-- Ascends to a safe [return altitude](#return_altitude) (above any expected obstacles).
-- Flies via direct path to a rally point or the start of a [mission landing pattern](#mission_landing_pattern) (whichever is closest).
+- Ascends to a safe [minimum return altitude](#minimum-return-altitude) (above any expected obstacles) if needed.
+- Flies via direct path to a rally point or the start of a [mission landing pattern](#mission-landing-pattern) (whichever is closest).
   If no mission landing or rally points are defined the vehicle instead returns home via a direct path.
 - If the destination is a mission landing pattern it will follow the pattern to land.
-- If the destination is a rally point or home it will [land or wait](#arrival) at descent altitude (depending on landing parameters).
-
-<a id="mission_landing_pattern"></a>
-
-:::note
-A mission landing pattern consists of a [MAV_CMD_DO_LAND_START](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_LAND_START), one or more position waypoints, and a [MAV_CMD_NAV_LAND](https://mavlink.io/en/messages/common.html#MAV_CMD_NAV_LAND).
-:::
+- If the destination is a rally point or home it will [land or wait](#hover-landing-at-destination) at descent altitude (depending on landing parameters).
 
 :::warning
-When this type is set PX4 will reject any mission without a valid landing pattern.
+When this type is set PX4 will reject a mission without a valid landing pattern.
```

Comment:
@sfuhrer I wanted to clarify the VTOL Return mode landing behaviour for RTL_TYPE=1 (the default), and specifically this point "When this type is set PX4 will reject a mission without a valid landing pattern.".

For a plane this makes sense. You probably won't use a rally point because there is no way to specify the flight path to land. So you define a mission and a mission landing pattern, and you always land on it whether in a mission or return mode.

But what about VTOL? I just tried this and I was forced to define a "fixed wing" landing pattern. But a VTOL can land vertically - it doesn't need to define the flight path. Now I could define rally points and all should work well, but I am STILL forced to define the mission landing pattern, and if I actually want to have a mission, I'm going to have to land using this pattern as though I was fixed wing. What's up?